javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,205 @@
+/* 
+        I did not finish yet 
+
+*/
```

Comment:
Wow, the real voice assistant app!?
The idea of this homework was to make sure you understand the basics of JavaScript and feel comfortable with it, and I think you got it:)
In general, I'd try to focus on the basics, and then on adding extra features, like `SpeechRecognition`. But well done!